Code/Engine/PlayerController.cpp

```diff
@@ -18,7 +18,7 @@ sad::PlayerController::~PlayerController() {}
 void sad::PlayerController::Update()
 {
 	
-	if (InputManager::GetInstance().GetAxis(SDL_CONTROLLER_AXIS_LEFTX) != 0.0f);
+	InputManager& input = InputManager::GetInstance();
 
 	// Movespeed, to later get on config instead.
```

Comment:
Unless I misunderstand it - this comment can probably be removed for now. I don't think we'll have the movement speed in a config file?